### PR TITLE
Automatic fetching sprint videos from youtube playilist

### PR DESCRIPTION
**Pull request by xeviknal**
Goal: Automatically update the sprint-videos section in kiali.io.

If we upload the sprint demo on youtube before the automatic release deploy is performed (Friday EOD), then Kiali.io will automatically fetch and build with that video.

As it is right now, we should have to manually change the page to add the video. This section is currently updated and shows some gaps between videos.

Pre-requisite before merge this video:
1) Add the latests sprint demo videos into the [Sprint demos playlist](https://www.youtube.com/playlist?list=PLgwrCYrAkQYOHIeGukEZUmNRGbqI9uwpi). The changes proposed here are displaying only videos in that playlist.
2) Have an agreement that we will add upcoming videos into that playlist.
3) Have an agreement to name the videos on Youtube in standardized way making sure the release version is in the title.

I do also volunteer to add the latest demo vids to the playlist. However, I'd need privileges to manage the kiali channel in YT.

### CODE REVIEW

**Review by jshaughn — APPROVED** [review]
@xeviknal This will be nice if it works, we definitely have too many manual updates required for the website.  I'll approve now although see me comment about video titles.

File: themes/kiali/layouts/partials/home/video-sprint-demo-section.html [hunk 1]
```diff
@@ -1,55 +1,40 @@
...
         </span>
     </h1>
 
     <div id="sprint-videos" class="new-embed-video-links">
-        <div class="video-card">
-            <header class="video-header">
-                <span class="video-text">September 18, 2020</span>
-                <span class=" video-text video-icon"><span class="fas fa-video"/></span>
-            </header>
-            <p class="video-text video-title">Sprint 45 [Kiali v1.24]</p>
```
[the remark this answers is not in the document]
Comment (xeviknal) [reply themes/kiali/layouts/partials/home/video-sprint-demo-section.html:18]: +100. I completely agree with that one. I think we should also add more keywords in the title too. To have more presence out there and capture people on youtube's feeds.

**Comment by israel-hdez** [comment]
@xeviknal I understand this is still not fully automated, right?
We still would need to somehow manually trigger a build -- similarly to how we need to trigger a build if we want to refresh the "latest in our blog".

But, honestly, this is a big step forward :+1:

**Comment by xeviknal** [comment]
@israel-hdez yep, since hugo is for static web pages, we only can add logic in the building time.
I didn't considered to add  jQuery or another framework to fetch the vids in client side. But maybe next time!

The good point in this scenario is that if we add the video in youtube right after the meeting, the release deploy at EOD will update the section. So the section will be updated (most of the times, human hand involved still!).

btw, what time is the release deploy at? Friday evening? Saturday?

**Comment by israel-hdez** [comment]
> btw, what time is the release deploy at? Friday evening? Saturday?

It's Friday afternoon. About 3-4PM...

**Pull request merged**